src/features/Dashboard/DashboardPage/index.scss

```diff
@@ -23,6 +23,13 @@
   min-height: 426px;
 }
 
+.truncated-text {
+  width: 250px;
```

Comment:
Shouldn't be a fixed width because in the case the section is less of that amount it doesn't look good, should be a percentage
![Screenshot from 2024-10-22 20-04-20](https://github.com/user-attachments/assets/6b3711c6-8f91-4717-a227-b220022c9faf)